mist-core/src/main/java/edu/snu/mist/core/replay/EventReplayUtils.java

```diff
@@ -0,0 +1,292 @@
+import org.json.simple.parser.JSONParser;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.*;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * This class is used for replaying events with the replay server.
+ * It sends REST API calls to the replay server.
+ * The server must support these functions accordingly.
+ */
+public final class EventReplayUtils {
+
+  private static final Logger LOG = Logger.getLogger(EventReplayUtils.class.getName());
+
+  private EventReplayUtils() {
+    // empty constructor
+  }
+
+  /**
+   * Sends a POST to make an mqtt connection and subscribe request topic.
+   * This method is the equivalent to :
+   * > curl -X POST http://(replayAddress):(replayPort)/subscribe \
+   *     -H 'Content-Type: application/json' \
+   *     -d '{
+   *        "brokerAddress": "(brokerAddress)",
+   *        "brokerPort": (brokerPort),
+   *        "topic": "(topic)"
+   *        }'
+   * @return true on success, else false
+   */
+  public static boolean subscribe(final String replayAddress, final int replayPort,
+                                  final String brokerAddress, final int brokerPort, final String topic) {
+    try {
+      final URL url = new URL(getReplayServerUrl(replayAddress, replayPort) + "/subscribe");
+      final HttpURLConnection conn = (HttpURLConnection) url.openConnection();
+      conn.setRequestMethod("POST");
+      conn.setRequestProperty("Content-Type", "application/json");
+      conn.setDoOutput(true);
+
+      final StringBuilder requestBody = new StringBuilder("{");
+      requestBody.append("\"brokerAddress\": \"" + brokerAddress);
+      requestBody.append("\", \"brokerPort\": " +  brokerPort);
+      requestBody.append(", \"topic\": \"" + topic + "\"}\n");
+
+      final OutputStreamWriter writer = new OutputStreamWriter(conn.getOutputStream(), "UTF-8");
+      writer.write(requestBody.toString());
+      writer.flush();
+      writer.close();
+
+      if (conn.getResponseCode() != 200) {
+        LOG.log(Level.WARNING, "Failed : HTTP error code is {0}. Failure Response message : {1}",
+            new Object[]{conn.getResponseCode(), conn.getResponseMessage()});
+
+        // End the connection.
+        conn.disconnect();
+        return false;
+      }
+
+      // End the connection.
+      conn.disconnect();
+      return true;
+    } catch (Exception e) {
+      e.printStackTrace();
+      return false;
+    }
+  }
+
+  /**
+   * Sends a GET to retrieve data within a timestamp period.
+   * This method is the equivalent to :
+   * > curl -X GET http://(replayAddress):(replayPort)/replay \
+   *     -H 'Content-Type: application/json' \
+   *     -d '{
+   *        "brokerURI": "(brokerURI)",
+   *        "topic": "(topic)"
+   *        }'
+   */
+  public static EventReplayResult replay(final String replayAddress,
```

Comment:
The parameter names are confusing.
`replayAddress` -> `replayServerAddress`
`replayPort` -> `replayServerPort`